dix/req.c

```diff
@@ -1806,21 +1808,29 @@ static int dix_cas_rops_send(struct m0_dix_req *req)
 
 			switch (req->dr_type) {
 			case DIX_GET:
-				rc = m0_cas_get(creq, &cctg_id,
-						&cas_rop->crp_keys);
+				rc = (req->dr_is_meta ?
```

Comment:
Why cannot we use versioning on meta indexes?

Reply:
The main issue was enabling `COF_OVERWRITE` on put, which ended up changing the semantics a bit. Some of the tests or setup code expects that creating an index that already exists will return `EEXIST`, but this change allows silent overwrites. Since the initial quorum functionality is focusing on record operations only, I just left versioning off for meta indices.

Reply:
I see, thanks.